clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/provider/agent/GoogleNetworkLoadBalancerCachingAgent.groovy

```diff
@@ -45,7 +45,7 @@ import static com.netflix.spinnaker.clouddriver.google.cache.Keys.Namespace.INST
 import static com.netflix.spinnaker.clouddriver.google.cache.Keys.Namespace.LOAD_BALANCERS
 
 @Slf4j
-class GoogleLoadBalancerCachingAgent extends AbstractGoogleCachingAgent implements OnDemandAgent {
+class GoogleNetworkLoadBalancerCachingAgent extends AbstractGoogleCachingAgent implements OnDemandAgent {
```

Comment:
I'll close this PR and split this out into 

1. Caching, including adding the `GoogleHttpLoadBalancer` objects.
2. Refactor.